app/javascript/app/providers/metadata-provider/metadata-provider-actions.js

```diff
@@ -10,7 +10,7 @@ export const fetchMetaFail = createAction('fetchMetaFail');
 function getDataByMeta(meta) {
   switch (meta) {
     case 'ghg':
-      return SAAPI.get('emissions/meta');
+      return SAAPI.get('emissions/meta', { deeply_nested_sectors: false });
```

Comment:
@Bluesmile82 I'm not sure if we need to filter metadata too. Do we need to list all sectors from metadata anywhere?

Reply:
I don't think we need to list all the sectors, so I'd go with filtering it on the metadata as well. We only need to download all data (when downloading), so no filtering on that one.)

Reply:
:+1: